base/src/main/scala/org/bykn/bosatsu/Macro.scala

```diff
@@ -0,0 +1,20 @@
+package org.bykn.bosatsu
+
+import reflect.macros.blackbox.Context
+import scala.io._
+
+import scala.util.{Try => STry}
+
+class Macro(val c: Context) {
+  import c._, universe._
+  def smac(file: c.Expr[String]) = file.tree match {
+    case Literal(Constant(s: String)) =>
+      val res = STry(
+        Source.fromFile(s, "UTF-8").getLines.mkString("\n")
+      ).getOrElse(
+        Source.fromFile(s"external/org_bykn_bosatsu/$s", "UTF-8").getLines.mkString("\n")
```

Comment:
I added this in just to get things working. We only end up over here when using `rules_bosatsu`. But I can't figure out why.

Reply:
The path to the file is different evidently in bazel and sbt. That’s a bummer.

Reply:
The path to the file is different evidently in bazel and sbt. That’s a bummer.

Reply:
So the weird part is this works in bazel with `bazel build //cli/...` and `bazel build //core/...`... just not with `bazel build //test_workspace/...`

Reply:
Bazel puts external repositories on a different path. It was going to be fixed like 2 years ago and as far as I can tell they just didn’t bother. The test_workspace configured a new workspace to exercise a remote repo just for this reason.

There is a work around but I forget what it is at the moment. Ugh.

Reply:
Ohhh... Is that what the resources_path parameters are all about?